### PR TITLE
add callpath example

### PR DESCRIPTION
**Pull request by marosiag**
add callpath example

### CODE REVIEW

File: apax.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: "ax-download-example"
-version: 1.0.0
+version: 2.0.0
 type: app
 targets:
   - "1500"
   - plcsim
 devDependencies:
-  "@ax/sdk": 3.0.17
+  "@ax/sdk": ^5.0.0
```
Comment (sjuergen) [inline apax.yml]: please pin the version to a 5.x.y

File: src/configuration.st [hunk 2]
```diff
@@ -2,12 +2,23 @@ CONFIGURATION PLC_1
     VAR_GLOBAL
         GlobalInt : int;
```
Comment (sjuergen) [inline src/configuration.st]: please all keywords in capital letters 

Int, int --> INT
Global variables should start with capital letters. Please refer ton the style guide at our AX documentation

File: src/callpathExample/checks.st [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FUNCTION checkHundredth : BOOL
```
Comment (sjuergen) [inline src/callpathExample/checks.st]: Function start with capital letters

Comment (sjuergen) [reply src/callpathExample/checks.st]: VAR_INPUTs --> Start with capital letter

File: src/callpathExample/callpathExample.st [hunk 4]
```diff
@@ -0,0 +1,29 @@
+PROGRAM UpCounter
+    VAR_EXTERNAL
+        counterInt:       Int;
```
Comment (sjuergen) [inline src/callpathExample/callpathExample.st]: CounterInt : INT --> style guide conform

File: apax.yml [hunk 5]
```diff
@@ -1,11 +1,11 @@
 name: "ax-download-example"
```
Comment (sjuergen) [inline apax.yml]: ae-debugging

File: src/configuration.st (hunk 2 above)
Comment (marosiag) [reply src/configuration.st]: done

File: apax.yml (hunk 5 above)
Comment (marosiag) [reply apax.yml]: good catch! done

File: src/callpathExample/callpathExample.st (hunk 4 above)
Comment (marosiag) [reply src/callpathExample/callpathExample.st]: style guide applied - not only for this one

**Review by sjuergen — APPROVED** [review]

**Pull request merged**